src/js/controller/controller.js

```diff
@@ -263,6 +263,10 @@ define([
 
                 _checkAutoStart();
                 _model.change('viewable', viewableChange);
+                _model.change('viewable', _checkPlayOnViewable);
+                _model.on('change:autostartFailed change:autostartMuted change:mute', function(model) {
+                    model.off('change:viewable', _checkPlayOnViewable);
+                });
```

Comment:
@egreaves I moved the removal of the listener to controller.js because _checkPlayOnViewable is undefined in controls.js